language/anope.it_IT.po

```diff
@@ -9085,3 +9054,66 @@ msgstr "{nick | canale}"
 
 msgid "{nick | channel} memo-text"
 msgstr "{nick | canale} testo-memo"
+
```

Comment:
Can you delete the old strings from the bottom of the file?

Reply:
Done :)